org.goldcoin.goldcoin-qt.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "org.goldcoin.goldcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "goldcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "goldcoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/goldcoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
```

Comment:
This doesn't do what you think it does. you still build an aarch64 flatpak...

Reply:
I believe that aarch64 build requires ["aarch64"] not ["x86_64"]. Have your system does not recognize the difference between these two architecture?

Reply:
This doesn't do what you think it does. 

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Of course you could build from source instead.